resources_pages/block_rep.md

```diff
@@ -0,0 +1,23 @@
+# Block Representative Guidelines
+
+To assist in this, every block, a student volunteers to represent their peers, acting as a liaison between students and MDS leadership (we/our/ours). This allows us to see how the block is going, allowing us to make changes if need be. 
```

Comment:
What is "this"? I'm a bit lost.

Reply:
Oops. I had deleted a previous sentence!